include/open62541/plugin/eventloop.h

```diff
@@ -750,6 +750,13 @@ UA_ConnectionManager_new_MQTT(const UA_String eventSourceName);
 UA_EXPORT UA_InterruptManager *
 UA_InterruptManager_new_POSIX(const UA_String eventSourceName);
 
+#elif defined(UA_ARCHITECTURE_ZEPHYR)
```

Comment:
This header is for the plugin interface definition.
Please move the plugin implementation to /plugins/include/...

Reply:
Scratch that. The /arch things are correct in this header.